lib/calculate-class-breaks/calculate-std-dev-intervals.js

```diff
@@ -0,0 +1,30 @@
+const ss = require('simple-statistics')
+
+function calculateStdDevIntervals (values, classification) {
+  if (!classification.stddev_intv) throw new Error('must supply a standard deviation interval')
+  const intv = classification.stddev_intv
+  if (intv !== 0.25 && intv !== 0.33 && intv !== 0.5 && intv !== 1) throw new Error('Unacceptable interval value: ' + intv)
```

Comment:
I seem unable to find where the `classification.stddev_intv` is used in this PR. Can you point it out?

Reply:
Nice catch!  The legacy code doesn't actually seem to use this either.  Perhaps it was used at one point.  I will remove.

Reply:
Done.